src/compose.py

```diff
@@ -19,7 +19,8 @@ def pull(settings):
 
 def run(args, settings):
     docker_prefix = settings["docker_prefix"]
-    prefix = 'docker-compose --project-name {} '.format(docker_prefix)
+    staging_file =  "-f ../docker-compose.staging.yml" if settings["fake_smtp"] else ""
```

Comment:
This seems to be the way to conditionally include containers using compose, by having them in separate file which yoi spin up together.